master/reference/calicoctl/resources/felixconfig.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Felix Configuration Resource (FelixConfiguration)
+---
+
+A Felix configuration resource (FelixConfiguration) represents Felix configuration options for the cluster.
+
+For `calicoctl` commands that specify a resource type on the CLI, the following
+aliases are supported (all case insensitive): `felixconfiguration`, `felixconfig`, `felixconfigurations`, `felixconfigs`.
+
+See [Configuring Felix]({{site.baseurl}}/{{page.version}}/referance/felix/configuration) for more details.
+
+### Sample YAML
+
+```yaml
+apiVersion: projectcalico.org/v2
+kind: FelixConfiguration
+metadata:
+  name: default
+spec:
+  ipv6Support: false
+  ignoreLooseRPF: true
+  chainInsertMode: Append
+```
+
+### Felix Configuration Definition
+
+#### Metadata
+
+| Field       | Description                 | Accepted Values   | Schema |
+|-------------|-----------------------------|-------------------|--------|
+| name     | Unique name to describe this resource instance. Required. | Alphanumeric string with optional `.`, `_`, `-`, or `/` | string |
+
+- Resource with the name "default" has a specific meaning - this contains the Felix global default configuration.
+- Resource with the name "node.<nodename>" has the specific meaning that this contains the node-specific overrides, and will be applied to the node <nodename>.  When deleting a node the FelixConfiguration resource associated with the node will also be deleted.
```

Comment:
"Resource" --" Resources"
"has" --> "have"
(for grammar reasons..."Resource" would need an article here, but I think switching to a plural will be better in any case)
Also, you could delete "the specific meaning that this" and change "contains" to "contain"